.github/workflows/deployStg.yml

```diff
@@ -81,6 +81,45 @@ jobs:
           client_tarball: client.tgz
           deploy_env: ${{ env.DEPLOY_ENV }}
 
+  smoke_test_front_and_back_end:
+    runs-on: ubuntu-latest
+    needs: [ deploy ]
+    if: ${{ needs.deploy.result == 'success' }}
+    environment:
+      name: stg
+      url: https://stg.simplereport.gov
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{env.NODE_VERSION}}
+      - name: Cache yarn
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Set up dependencies
+        working-directory: frontend
+        run: yarn install --prefer-offline
+      - name: Smoke test the env
+        uses: ./.github/actions/post-deploy-smoke-test
+        with:
+          base_domain_name: stg.simplereport.gov
+
+  failed_smoke_test_pagerduty_alert:
+    runs-on: ubuntu-latest
+    if: ${{ ! needs.smoke_test_front_and_back_end.result == 'success' }}
+    needs: [ smoke_test_front_and_back_end ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Trigger PagerDuty alert
+        uses: ./.github/actions/pagerduty-alert
+        with:
+          summary: ":warning: Post-deploy smoke test couldn't verify that the frontend is talking to the backend. ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }} :warning:"
+          severity: error
+          routing_key: ${{ secrets.pagerduty_stg_integration_key }}
```

Comment:
I have a dumb question - Why do we need distinct keys for staging / dev / prod?  Aren't we just integrating one thing (github actions) with one other thing (pager duty)?  These keys aren't correlated to our actual environments / azure are they?

Reply:
These keys aren't correlated to our azure environments, but they are correlated to 3 distinct PagerDuty services.

- `pagerduty_dev_integration_key` connects to PagerDuty's [SimpleReport - Non-Prod](https://prime-cdc.pagerduty.com/service-directory/PTMCD3S) service
- `pagerduty_stg_integration_key` connects to PagerDuty's [SimpleReport - Stg](https://prime-cdc.pagerduty.com/service-directory/PRIUTXG) service
- `pagerduty_prod_integration_key` connects to PagerDuty's [SimpleReport - Production](https://prime-cdc.pagerduty.com/service-directory/P8M1OC7) service

Reply:
TIL, thanks!